inclusion/commitment.go

```diff
@@ -3,7 +3,6 @@ package inclusion
 import (
 	"crypto/sha256"
 
-	ns "github.com/celestiaorg/go-square/namespace"
 	sh "github.com/celestiaorg/go-square/shares"
```

Comment:
[no change needed] in a follow up we could remove the import alias `sh` here b/c IMO `shares.foo` is preferable over `sh.foo`. `shares` won't conflict with any other import statements in this file so the type alias `sh` is unnecessary.